src/gui/guiChatConsole.cpp

```diff
@@ -619,6 +620,13 @@ bool GUIChatConsole::OnEvent(const SEvent& event)
 			m_chat_backend->scroll(rows);
 		}
 	}
+#if defined(IRRLICHT_VERSION_MT_REVISION) && (IRRLICHT_VERSION_MT_REVISION >= 2)
```

Comment:
Since this is client code you can count on `IRRLICHT_VERSION_MT_REVISION` being defined.

Reply:
Weirdly, the CI did complain about the missing string input events in the chat console and the edit boxes